server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/AutoDbMigration.java

```diff
@@ -75,6 +77,27 @@ void installH2() {
   @VisibleForTesting
   protected void createSchema(Connection connection, String dialectId) {
     DdlUtils.createSchema(connection, dialectId, false);
+    hackFixForProjectMeasureTreeQueries(connection);
```

Comment:
This fix should be applied on H2 only. Maybe a link to the JIRA ticket could help to understand the hack.